service/src/main/java/bio/terra/externalcreds/models/TokenTypeEnum.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.externalcreds.models;
+
+public enum TokenTypeEnum {
+  DOCUMENT,
+  ACCESS
```

Comment:
these don't match the database enum `CREATE TYPE token_type_enum AS ENUM ('document_token', 'access_token')`